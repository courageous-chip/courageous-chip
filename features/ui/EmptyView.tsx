import React, { FC } from "react";
import { StyleSheet, Text, View } from "react-native";

export const EmptyView: FC = function () {
  return (
    <View style={styles.container}>
      <Text style={styles.text}>ü§∑‚Äç‚ôÄÔ∏è</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    alignItems: "center",
    backgroundColor: "#fff",
    flex: 1,
    justifyContent: "center",
  },
  text: {
    fontSize: 72,
  },
});
