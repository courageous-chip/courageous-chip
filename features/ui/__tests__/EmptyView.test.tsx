import React from "react";
import renderer from "react-test-renderer";

import { EmptyView } from "../EmptyView";

describe("<EmptyView />", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<EmptyView />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
