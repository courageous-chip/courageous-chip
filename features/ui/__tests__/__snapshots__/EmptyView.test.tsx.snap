// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptyView /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 72,
      }
    }
  >
    🤷‍♀️
  </Text>
</View>
`;
